inc/composer/class-plugin.php

```diff
@@ -43,7 +43,15 @@ public function install_files() {
 		// Update the .gitignore to include the wp-config.php, WordPress, the index.php
 		// as these files should not be included in VCS.
 		if ( ! file_exists( $dest . '/.gitignore' ) ) {
-			file_put_contents( $dest . '/.gitignore', "wordpress/\nindex.php\nwp-config.php" );
+			$entries = [
+				'# Altis',
+				'wordpress',
+				'index.php',
+				'wp-config.php',
+				'chassis*',
```

Comment:
I would change this to just `chassis` for now, and we can always change it if we do decide to rename the directories.

Reply:
Alrighty then